### PR TITLE
#175 Updated license in poms

### PR DESCRIPTION
**Pull request by SmiddyPence**
Review and pull please

### CODE REVIEW

File: javaparser-core/pom.xml [hunk 1]
```diff
@@ -1,22 +1,22 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ftomassetti) [inline javaparser-core/pom.xml:1]: Why has been this line removed here and in the other pom.xml?

**Comment by ftomassetti** [comment]
It looks good to me, I would just leave the line specifying the encoding

File: javaparser-core/pom.xml (hunk 1 above)
Comment (SmiddyPence) [inline javaparser-core/pom.xml:1]: Convention is not to include it, was only present in one of our current poms.

Typically license doesn't appear either though.

Comment (ftomassetti) [inline javaparser-core/pom.xml:1]: My bad, I thought it was suggested to include it

**Comment by ftomassetti** [comment]
There is also the `licenses` tag in the main pom to be updated

**Comment by SmiddyPence** [comment]
@ftomassetti I have changed this to use the `<license>` block instead.  I couldn't find examples where verbatim is the pom like we have.

**Comment by ftomassetti** [comment]
Makes sense to me, it should be easier to maintain also and harder to get unnoticed because it appears on Maven central

**Pull request merged**